src/types/types.ts

```diff
@@ -813,6 +830,71 @@ export interface GetFloorPriceResponse {
   readonly looksRare: FloorPriceMarketplace | FloorPriceError;
 }
 
+/**
+ * Optional parameters object for the the {@link getContractsForOwner} method
+ *
+ * @public
+ */
+export interface GetContractsForOwnerOptions {
+  /** Key for pagination to use to fetch results from the next page if available. */
+  pageKey?: string;
+
+  /**
+   * Optional list of filters applied to the query. NFTs that match one or more
+   * of these filters are included in the response. May not be used in
+   * conjunction with {@link excludeFilters}.
+   */
+  includeFilters?: NftFilters[];
+
+  /**
+   * Optional list of filters applied to the query. NFTs that match one or more
+   * of these filters are excluded from the response. May not be used in
+   * conjunction with {@link includeFilters}
+   */
+  excludeFilters?: NftFilters[];
+}
+
+/**
+ * The response for the {@link NftNamespace.getContractsForOwner} method.
+ *
+ * @public
+ */
+export interface GetContractsForOwnerResponse {
+  /** The list of contracts, that match the query, held by the given address. */
+  contracts: ContractForOwner[];
+
+  /** Key for pagination to use to fetch results from the next page if available. */
+  pageKey?: string;
+
+  /** Total number of NFT contracts held by the given address. */
+  totalCount: number;
+}
+
+/** Represents a single NFT contract data in the {@link GetContractsForOwnerResponse}. */
+export interface ContractForOwner extends Omit<NftContract, 'totalSupply'> {
```

Comment:
There's no need to omit `totalSupply` since it's included in the response. Good eye on catching this though! I've let the docs team know to update the API docs.

Reply:
Oh ok, let me fix that real quick. Yeah, I actually used the docs to implement the feature and didn't notice in the response that field was there. Thank you for letting me notice that.

Reply:
Oh ok, let me fix that real quick. Yeah, I used the documentation to shape the response object, my bad for not noticing that the field was there.